idaes/models/properties/general_helmholtz/helmholtz_functions.py

```diff
@@ -481,6 +634,31 @@ def _sv_str(**kwargs):
     def add_funcs(self, names=None):
         add_helmholtz_external_functions(self.blk, names=names)
 
+    def _state_vars(self, **kwargs):
```

Comment:
Could you add a docstring describing what's supposed to be returned here? It's hard to follow the code further on without knowing what "c", "u", etc. are.

Reply:
Yes